app/models/manageiq/providers/amazon/cloud_manager.rb

```diff
@@ -47,6 +47,10 @@ class ManageIQ::Providers::Amazon::CloudManager < ManageIQ::Providers::CloudMana
   supports :regions
   supports :discovery
 
+  def ensure_network_manager
```

Comment:
Hmm, where is this called from? 
Isnt the `ensure_managers`  method where this should be?